### DISCUSSION TITLE
waitForTransactionReceipt returns error

### DISCUSSION THREAD

**Discussion by FreezyEx** [wevm/viem#481] [Question]
I am trying to get the tx response using `waitForTransactionReceipt`. I send the transaction, get the hash as response and pass this hash to `waitForTransactionReceipt`. The error I get is:

`Transaction with hash "0x44cd834..." could not be found.`

**Comment by firnprotocol** [comment]
i believe i got this once as well; @tmm, we were just talking about this.

except i used wagmi's [`waitForTransaction`](https://wagmi.sh/core/actions/waitForTransaction) instead—but they should be similar.

**Comment by FreezyEx** [comment]
Seems that has been fixed with the last updates.

Il Dom 21 Mag 2023, 17:06 Firn Protocol ***@***.***> ha
scritto:
